DEPLOYING.md

```diff
@@ -125,3 +174,62 @@ To check the logs for a specific service:
 sudo docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-theme.yaml logs -f ckan
 ```  
 *(exit the logs by pressing Ctrl+C at any time)*
+
+To get inside a running container
+
+```
+sudo docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-theme.yaml exec {service-name} bash
```

Comment:
Do all of them have `bash` executable? I think there's one (forgot which) that only has `sh`.